### ISSUE TITLE
asp.net application hangs/crash using glimpse, iis hangs

### ISSUE THREAD

**Issue by aprjanka** [Glimpse/Glimpse#787]
My first time experience with glimpse (asp.net mvc5 nuget package) is a complete disaster.
I turned on diagnostic logging, and found the following errors:
1.
ERROR | Exception executing tab of type 'glimpse_metadata'. | System.MissingMethodException--Constructor on type (some typeconverter) not found.

It seems glimpse try to create one of my typeconverter with the empty constructor but it has no empty constructor. 
1. Tons of "ERROR | Error serializing object. | Newtonsoft.Json.JsonSerializationException"

Why glimpse try to serialize models???? This is very stupid idea, what about circular references? complex properties which shouldn't be serialized ever?

How can we fix this issues?

**Comment by aprjanka** [comment]
Tried again the latest version. Same problem. IIS worker process hangs forever, almost on every page.

I found that the Cache tab is the problem. Glimpse try to serialize (why?) my cache objects with Newtonsoft json serializer. If the json serialization fails for any reason glimpse try again in an infinite loop and the web application hangs.

Please fix this.

**Comment by DanDiplo** [comment]
I've just experienced this issue. IIS was inexplicably crashing. I narrowed it down to a particular Entity Framework 5 query that, if removed, prevented the crash. But I couldn't understand why it was happening. Just by chance I disabled Glimpse and the crash went away. I then tried disabling every tab in Glimpse until I discovered it was the MetaData tab. Once disabled the crash went away.

This is using latest Glimpse 1.8.6
